accounting/filters/OsgScheddCpuRemovedFilter.py

```diff
@@ -0,0 +1,400 @@
+                data["NumJobStarts"],
+                data["NumShadowStarts"],
+                data["_NoShadow"]):
+            if no_shadow:
+                num_exec_attempts.append(None)
+                num_shadow_starts.append(None)
+            else:
+                num_exec_attempts.append(job_starts)
+                num_shadow_starts.append(shadow_starts)
+
+        # Short jobs are jobs that ran for < 1 minute
+        is_short_job = []
+        for (goodput_time, record_date, start_date) in zip(
+                data["CommittedTime"],
+                data["RecordTime"],
+                data["JobCurrentStartDate"]):
+            if (goodput_time is not None) and (goodput_time > 0):
+                is_short_job.append(goodput_time < 60)
+            elif None in (record_date, start_date):
+                is_short_job.append(None)
+            else:
+                is_short_job.append((record_date - start_date) < 60)
+
+        # "Long" (i.e. "normal") jobs ran >= 1 minute
+        # We only want to use these when computing percentiles,
+        # so filter out short jobs and removed jobs,
+        # and sort them so we can easily grab the percentiles later
+        long_times_sorted = []
+        for (is_short, goodput_time, no_shadow, job_status) in zip(
+                is_short_job,
+                data["CommittedTime"],
+                data["_NoShadow"],
+                data["JobStatus"]):
+            if (is_short == False) and (no_shadow == False) and (job_status != 3):
+                long_times_sorted.append(goodput_time)
+        long_times_sorted = self.clean(long_times_sorted)
+        long_times_sorted.sort()
+
+        # Compute columns
+        row["All CPU Hours"]    = sum(self.clean(total_cpu_time)) / 3600
+        row["Good CPU Hours"]   = sum(self.clean(goodput_cpu_time)) / 3600
+        row["Num Uniq Job Ids"] = sum(data['_NumJobs'])
+        row["Num Rm'd Jobs"]    = sum([status == 3 for status in data["JobStatus"]])
+        row["Num Jobs Rm'd w/o exec"]= sum([starts == 0 for starts in data["NumJobStarts"]]) 
+        row["Avg MB Sent"]      = stats.mean(self.clean(data["BytesSent"], allow_empty_list=False)) / 1e6
+        row["Max MB Sent"]      = max(self.clean(data["BytesSent"], allow_empty_list=False)) / 1e6
+        row["Avg MB Recv"]      = stats.mean(self.clean(data["BytesRecvd"], allow_empty_list=False)) / 1e6
+        row["Max MB Recv"]      = max(self.clean(data["BytesRecvd"], allow_empty_list=False)) / 1e6
+        row["Num Short Jobs"]   = sum(self.clean(is_short_job))
+        row["Max Rqst Mem MB"]  = max(self.clean(data['RequestMemory'], allow_empty_list=False))
+        row["Med Used Mem MB"]  = stats.median(self.clean(data["MemoryUsage"], allow_empty_list=False))
+        row["Max Used Mem MB"]  = max(self.clean(data["MemoryUsage"], allow_empty_list=False))
+        row["Max Rqst Cpus"]    = max(self.clean(data["RequestCpus"], allow_empty_list=False))
```

Comment:
Not important for this first version of the removed jobs report, but it would be nice to filter out jobs that removed without an execution attempt for these columns.